lib/addons/jupyterhub/index.ts

```diff
@@ -0,0 +1,52 @@
+import { Construct } from "constructs";
+import { ClusterInfo } from '../../spi';
+import { createNamespace } from '../../utils';
+import { HelmAddOn, HelmAddOnProps, HelmAddOnUserProps } from '../helm-addon';
+
+/**
+ * Configuration options for the add-on.
+ */
+ type JupyterHubAddOnProps = HelmAddOnUserProps;
+
+const JUPYTERHUB = 'jupyterhub';
+const RELEASE = 'blueprints-addon-jupyterhub';
+
+/**
+ * Defaults options for the add-on
+ */
+const defaultProps: HelmAddOnProps = {
```

Comment:
It would make sense to allow specifying storage class based on [this](https://zero-to-jupyterhub.readthedocs.io/en/latest/jupyterhub/customizing/user-storage.html#type-of-storage-provisioned).
Also how much [size](https://zero-to-jupyterhub.readthedocs.io/en/latest/jupyterhub/customizing/user-storage.html#size-of-storage-provisioned) is allocated.

Reply:
Addressed - this makes use of EBS volume with the CSI addon